test/ecpair.js

```diff
@@ -160,18 +160,6 @@ describe('ECPair', function () {
         var keyPair = ProxiedECPair.makeRandom()
         assert.strictEqual(keyPair.toWIF(), exWIF)
       })
-
-      it('passes the options param', sinon.test(function () {
```

Comment:
Removed this test,  as https://github.com/cjohansen/Sinon.JS/issues/613 doesn't look like it'll be fixed any time soon.
We already check that the parameters themselves are being used,  it would just have been an added precaution to verify the call behaviour.